### ISSUE TITLE
Changing WOLF_RENDER_NODE does not seem to work

### ISSUE THREAD

**Issue by whazor** [games-on-whales/wolf#49]
I tried setting `WOLF_RENDER_NODE` to `/dev/dri/renderD129` to use the GPU instead of Intel QuickSync. But it keeps using Intel and D128. 

I verified that the environment variable is correctly set by running `-ti ... bash` on the `docker run` command:
```
root@game-srv:/wolf# env | grep WOLF_RENDER_NODE
WOLF_RENDER_NODE=/dev/dri/renderD129
```

The way I found out it is using Intel is because of the HEVC encoder selection: `19:11:16.370946074 INFO  | Selected HEVC encoder: qsv`

**Comment by ABeltramo** [comment]
Have you seen [the docs](https://games-on-whales.github.io/wolf/stable/user/configuration.html#_multiple_gpu) on this?

TL;DR: 
 - `WOLF_RENDER_NODE` only controls on which GPU the virtual screen will be created
 - The encoder pipeline is controlled by the order of the entries in the `config.toml`. The default is: `nvidia` -> `quicksink` -> `vaapi` (both Intel and AMD) -> `x265` (software). The order in which they appear in the `config.toml` is the order in which they'll be tried by Wolf.

Which vendor is the GPU that you are trying to use?  
If it's Intel or AMD based on [this issue](https://gitlab.freedesktop.org/gstreamer/gstreamer/-/issues/1167) you should be able to replace `vah265enc` with `varenderD129h265enc` in `encoder_pipeline` under `[gstreamer.video.hevc_encoders]]` (and move that above the `qsv` entry or delete it). I don't have such a setup, so I haven't tested this personally.

I hope this helps!
